contracts/interfaces/IDCAStrategies.sol

```diff
@@ -304,8 +320,8 @@ interface IDCAStrategies is IDCAStrategiesManagementHandler, IDCAStrategiesPermi
   }
 
   struct ShareOfToken {
```

Comment:
Why are you doing this change? If you have `address[] addresses, uint80[] shares`, then you need to validate that `addresses.length == shares.length`. If you have `struct[]` then you don't need to validate it

Reply:
It's the only way I found to bypass `memory to storage not yet supported` error

Reply:
What you need to do is:
```
function createStrategy(
    string memory _strategyName,
    IDCAStrategies.ShareOfToken[] memory _tokens,
    address _owner
  ) external override returns (uint80 _strategyId) {
    bytes32 _key = ...
    for (uint i = 0; i < _tokens.length; ) {
        _tokenShares[key].push(_tokens[i]);
        unchecked {i++;}
    }
```